src/ui/components/NodePickerContext.tsx

```diff
@@ -153,7 +153,16 @@ export function NodePickerContextRoot({ children }: PropsWithChildren<{}>) {
     }
 
     // Start loading rects eagerly; we'll need them in the hover/click handlers
-    const boundingRectsPromise = boundingRectsCache.readAsync(replayClient, pauseId);
+    let boundingRectsPromise: PromiseLike<NodeBounds[]> | NodeBounds[] | undefined;
+    try {
+      boundingRectsPromise = boundingRectsCache.readAsync(replayClient, pauseId);
+    } catch {
+      setState({
+        status: "error",
+        options,
+      });
+      return;
+    }
```

Comment:
This reads weird, since catching a promise where it's defined (rather than awaited) won't do anything. Took me a minute to realize that you were catching here in case the cache had already resolved.

I think an easier-to-read way of accomplishing this same thing would be this though:
```suggestion
    boundingRectsCache.prefetch(replayClient, pauseId);
```

This method will start eagerly loading data, but suppress any thrown error.